### ISSUE TITLE
It looks like there is no functionality to parse RSQL to a constraint tree

### ISSUE THREAD

**Issue by dshefchik** [molgenis/molgenis-js-rsql#9]
### How to Reproduce
Look at src/

### Expected behavior
There would be a function to take an RSQL string and parse into a constraint tree

### Observed behavior
There is no function to take an RSQL string and parse into a constraint tree

**Comment by fdlk** [comment]
@dshefchik You're correct, the parser still lives in https://github.com/molgenis/molgenis-frontend/tree/master/packages/core-ui/src/modules/rest-client/rsql

**Comment by dshefchik** [comment]
Well that makes sense then! Might it make sense to move it into this repo?
